lib/mix/test/mix/utils_test.exs

```diff
@@ -49,30 +48,39 @@ defmodule Mix.UtilsTest do
   end
 
   test "symlink or copy" do
-    in_fixture "archive", fn ->
+    fun = fn ->
```

Comment:
Could you please revert this change and re-run the formatter using Elixir master? We have added `in_fixture` and `doctest` to the list of locals without parens based on your PR. Thank you!